opennlp-tools/src/main/java/opennlp/tools/cmdline/ArgumentParser.java

```diff
@@ -127,6 +127,21 @@ public Object parseArgument(Method method, String argName, String charsetName) {
     }
   }
 
+  private static class CharacterArgumentFactory implements ArgumentFactory {
+
+    public Object parseArgument(Method method, String argName, String argValue) {
```

Comment:
Is there any way that generics could be used with `ArgumentFactory` to return a `Character` instead of an `Object`? Guess that would be a larger refactor.